dtests/src/test/java/io/snappydata/hydra/consistency/mixedTableExtn.inc

```diff
@@ -0,0 +1,111 @@
+io.snappydata.hydra.testDMLOps.SnappySchemaPrms-snappyDDLExtn =
+"using column options(partition_by 'id', buckets '1', redundancy '1', persistent 'sync', COLUMN_MAX_DELTA_ROWS '12', COLUMN_BATCH_SIZE '1024', KEY_COLUMNS 'id')"
+"using row options(partition_by 'id', buckets '1', redundancy '1', persistent 'sync')"
+"using row options(persistent 'sync')"
+"using column options(partition_by 'id', buckets '1', redundancy '1', persistent 'sync', COLUMN_MAX_DELTA_ROWS '12', COLUMN_BATCH_SIZE '1024', KEY_COLUMNS 'id')"
+;
+
+io.snappydata.hydra.testDMLOps.SnappySchemaPrms-createTablesStatements =
+"create table app.table1 (id int, name varchar(20),code int, code1 double, code2 bigint, code3 varchar(30), tid int)"
```

Comment:
Change schema for tables. Add realistic schema if possible. Use complex types, dates, timestamp etc.